ci/.github/workflows/ci.yml

```diff
@@ -8,6 +8,8 @@ on:
   pull_request:
     branches:
       - '*'
+env:
+  RSPEC_CI: true
```

Comment:
Forces `rspec-rails` to depend on `4.0.0.pre`. See https://github.com/rspec/rspec-rails/blob/934276b0e12b21ad44bac54494c42f5a93e08a33/rspec-rails.gemspec#L48